src/websockets/binary_websockets.js

```diff
@@ -72,12 +72,22 @@ define(['es6-promise', 'reconnecting-websocket', 'js-cookie', 'token/token', 'jq
             delete unresolved_promises[key];
             if (data.error)
                 promise.reject(data.error);
+            else if (data.echo_req.trading_times) {
+                require(['common/menu'], function(menu) {
+                    markets = menu.extractMenu(data, {
+                        filter: function (sym) { return sym.feed_license !== 'chartonly'; }
+                    });
+                    promise.resolve(markets);
+                });
+            }
```

Comment:
what if some other module does not need the filter ** sym.feed_license !== 'chartonly'; ** ?